WordPress/build.gradle

```diff
@@ -642,10 +642,16 @@ tasks.register("dependencyTreeDiffCommentToGitHub", ViolationCommentsToGitHubTas
     commentTemplate = """
 ### The PR caused the following dependency changes:
 
+<details><summary>expand</summary>
+<p>
+
```

Comment:
That works for me, but I wonder if it wouldn't look nicer if instead of just `expand` as text here you'd put the section title directly as summary? Like:

```
<details><summary>The PR caused some dependency changes (expand to see details)</summary>
<p>
…
</p>
</details>
```

(In the end that's purely opinionated, so feel free to ignore)

Reply:
Sure, added in 1ef90d749a9dbae5dce5514afb1c0033908507ae